### DISCUSSION TITLE
DB is hanging while creating multiple proxies during initialization, if cache is enabled

### DISCUSSION THREAD

**Discussion by dtau00** [appy-one/acebase#158] [Q&A]
I posted this in one of the threads, but it might have gotten lost so creating a new thread.  I updated to the latest build (previously a month ago) after the fix with the index handling deleted nodes, and was having problems with the cache.  Everything works initially with the cache cleared. But when i log back in the second time, during my initialization process (where i create about a dozen proxies), somewhere in the middle the db hangs. Didnt' really see anything in the logs, just connection timeout errors. I have to restart the db and clear my cache. But with the cache disabled, everything has been working fine.

**Comment by appy-one** [comment]
Thanks, I'll setup a test for that to see what's going on

**Comment by appy-one** [comment]
I'm having a hard time reproducing this issue, none of my tests fail.

Can you provide the following info:
- are you running the client in the browser or Node.js?
- If you use Node.js, which version? If you run in the browser, which one and version?
- are you using bundlers such as rollup, webpack or browserify?
- are you including pre-bundled acebase packages from a CDN with a `script` include or are you using npm/yarn packages?
- Which versions of the following packages are you using:
   - acebase-server
   - acebase-client
   - acebase (for your cache db)
- How many proxies do you create?
- How large (KB/MB) is the data retrieved by the proxies in total?
- Is your server configured to use transaction logging?

Thanks!

**Reply by dtau00** [reply]
odd, appreciate you looking into it.  i'll do another check this weekend to see if i can narrow it down

**Reply by dtau00** [reply]
finally got around to checking on this.  maybe this will help someone in the future.  ended up being an issue with the transaction log.  not sure what exactly, but deleting it fixed it.
